src/Symfony/Contracts/HttpClient/Test/HttpClientTestCase.php

```diff
@@ -576,11 +576,11 @@ public function testResolve()
         ]);
 
         $this->assertSame(200, $response->getStatusCode());
-        $this->assertSame(200, $client->request('GET', 'http://symfony.com:8057/')->getStatusCode());
+        $this->assertSame(200, $client->request('GET', 'http://symfony.com:8057/', ['timeout' => $timeout = 0.1])->getStatusCode());
 
         $response = null;
         $this->expectException(TransportExceptionInterface::class);
-        $client->request('GET', 'http://symfony.com:8057/', ['timeout' => 3]);
+        $client->request('GET', 'http://symfony.com:8057/', ['timeout' => $timeout]);
```

Comment:
no need for the variable
let's use 1 to be sure we don't test a slow DNS resolver?

Reply:
The variable is to be sure that the value was high enough to succesfully resolve in the previous assertion. Logically, the second time should not take much longer (note that the value I put is 1000x the time needed (for safety)). What about $timeout * 2 to be sure ? 1 sec seems huge.

Reply:
I'm not sure 1s is that a big issue :)